test/run-tests.js

```diff
@@ -205,6 +205,48 @@ function run_compress_tests() {
                             failed_files[file] = 1;
                         }
                     }
+                    // The test with expect_stdout passed.
+                    // Now try to reminify original input with standard options
+                    // to see if it matches expect_stdout.
+                    var reminified_result = U.minify(test.input.print_to_string(), {
```

Comment:
Perhaps the input AST is the problem - it should have been cloned for later use by this code?

Reply:
May be try `input_code`?